vagrant-with-pbspro/hosts

```diff
@@ -0,0 +1,5 @@
+127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
+::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
+10.0.0.100  ood
+10.0.0.101  head
+10.0.0.102  pbscompute
```

Comment:
Both provide `10.0.0.102  pbscompute` but neither have it in their Vagrantfile. Is that a mistake or should there be a third node?